web/api/web_api_v1.c

```diff
@@ -915,6 +924,7 @@ static struct api_command {
         { "data",            0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_data            },
         { "chart",           0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_chart           },
         { "charts",          0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_charts          },
+        { "archivedcharts",  0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_archivedcharts  },
```

Comment:
What will this be used for, and how will it be used?

Reply:
The new endpoint to get archived charts is `/api/v1/archivedcharts`. You can query `127.0.0.1:19999/api/v1/archivedcharts` and get the list of charts that are archived. This is different than `/api/v1/charts` in that it only shows charts that no collector has touched yet. Those charts simply did not exist in the previous versions of netdata.

You can actually get the data of those charts by using the `/api/v1/chart?chart=name` endpoint.

Those 2 endpoints are the only way this increment can deliver value to the users.

Reply:
I don't think that end users will use the API. If I restart netdata can I see charts via the dashboard if they are archvied? Will they be visible to the cloud?

Reply:
The requirements stated that the only way to get the archived charts would be via a new API call. So they should not be visible in the dashboard. About ACLK I'm not sure what to do, it's related to @stelfrag 's comment. We should discuss this.

Reply:
I don't remember this point in the requirements - where is it (can I have a link?)